packages/varlet-ui/src/picker/props.ts

```diff
@@ -47,6 +48,16 @@ export const props = {
     type: [Number, String],
     default: 6,
   },
+  columnsNum: {
+    type: [String, Number],
+    validator(value: string | number) {
```

Comment:
no need this validator. You should process this prop in `computed` method

Reply:
```javascript
computed(() => {
if(...){
}
})
```

Reply:
```javascript
computed(() => {
    if(...){
        warn('Picker', '"columnsNum" should be > 0')
        return Infinity
    }
})
```
like this ?

Reply:
done-.-